musical_chairs/lib/musical_chairs.ex

```diff
@@ -1,2 +1,58 @@
 defmodule MusicalChairs do
+  @input_file "input.txt"
+  @output_file "output.txt"
+  
+  def main(_args) do
+    play_musical_chairs
+  end
+
+  defp play_musical_chairs do
+    @input_file
+    |> parse_rounds
+    |> play_rounds
+    |> write_results
+  end
+
+  defp write_results(rounds_results) do
```

Comment:
I'll typically order my functions roughly by how they are used. In this module, we are calling them roughly in order of `parse_rounds`, `play_rounds`, `write_results`. Helps the reader not have to hunt around the file so much.